### ISSUE TITLE
Export problem

### ISSUE THREAD

**Issue by roomdada** [Gustavinho/laravel-views#159]
The export does not take place after selection and click on the button of the grouped action. I can access the export class but the excel file is not downloaded. I need help urgently.
![Capture d’écran du 2022-09-07 10-29-26](https://user-images.githubusercontent.com/61591337/188858720-ab26412b-fd15-40f0-8221-7065e5ad8ac0.png)
![Capture d’écran du 2022-09-07 10-29-03](https://user-images.githubusercontent.com/61591337/188858727-7bf334cd-df15-427d-9293-130035a4bb79.png)
![Capture d’écran du 2022-09-07 10-28-43](https://user-images.githubusercontent.com/61591337/188858734-505942ad-d5a8-4c68-a1c0-7eed56a4c97a.png)

**Comment by Gustavinho** [comment]
Hi @roomdada, the returned value in the handler function is not being returned by the other internal functions, that's why you are not able to download the file.

I found a workaround for you, it's kind of tricky but as this is urgent for you, this is the fastest solution,  basically you will change the behavior of the internal `WithActions` trait

1. Copy this file somewhere in your project, https://github.com/Gustavinho/laravel-views/blob/master/src/Views/Traits/WithActions.php, remember to change the namespace
2. Change lines 31, 37 and, 54 adding a `return` statement
```php
31. return $this->executeActionHandler($action);
37. return $this->executeActionHandler($action);
54. return $action->handle($actionableItems, $this);
```
3. use this new trait in your view class, this will overwrite the original one

```php
use App\Your\Namespace\WithActions

use WithActions;
```

After that you should be able to download your file, please let me know how it goes or if you have a question about the workaround.

**Comment by Gustavinho** [comment]
You're almost there, it is just missing the return in the `handle` function of the action

**Comment by roomdada** [comment]
Always the same
![Capture d’écran du 2022-09-08 01-45-26](https://user-images.githubusercontent.com/61591337/189015261-38a16718-f74a-4d32-a0f8-ff6b6515fed0.png)

**Comment by roomdada** [comment]
Always the same
![Capture d’écran du 2022-09-08 01-45-26](https://user-images.githubusercontent.com/61591337/189015370-e105c2f9-c47d-4115-b978-b9ee8aa25fa6.png)

**Comment by Gustavinho** [comment]
Yeah it is not recommended, I just noticed you added the trait in the action class, try to to do it in the view class instead
